app/assets/stylesheets/controllers/teams.sass

```diff
@@ -25,4 +25,7 @@
 
 .icon-star-volunteer
   @include icon-before($star-empty)
-  color: $railsred
\ No newline at end of file
+  color: $railsred
+  
+.padding-dropdown 
```

Comment:
Can you please remove this class again? It's not used anymore and having unused code lying around is never good 😉